app/views/organizations/pets/show.html.erb

```diff
@@ -133,6 +116,12 @@
               </ul>
             </div>
           </div>
+          <%= link_to t('general.delete'), pet_path(@pet), class: 'btn btn-outline-dark mt-2',
```

Comment:
let's change this one to `btn-outline-danger` class

Reply:
Will do.